### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by mullenlucas**
Merging new-branch to main

### CODE REVIEW

File: file.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```
Comment (youmari) [inline file.html]: - [x] please make sure to add a good HTML structure where and add a head tag so you can link the CSS file with the HTML in order to change the color of the h1

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+A file that should not be uploaded
```
Comment (youmari) [inline test.md]: - [ ] please make sure to remove this file it should not be pushed to GitHub I encourage you to do that

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @mullenlucas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good commit messages


## Required Changes ♻️

- [x] please make sure to use the right linters files and delete the others and also make sure that they are passing 

- [x] You should add a Descriptive branch name, title, and short summary that tells in detail what you've done in Pull Request

- [x] Please make sure next time to use the[ rules ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)of writing Desctiprive commit messages in order for your PR will be pro 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: file.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse project</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```
Comment (Ugboaja-Uchechi) [inline file.html]: - [ ] [OPTIONAL] Kindly add a `class` attribute to the ` h1 ` tag and update your `CSS `to the `class` attribute according to the project requirement.

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @mullenlucas , 

## Status Approved  ✅  ✅ 

Great work fixing all the requirements stated by the previous code reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Your README looks professional ✅ 
- Linters are passing ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**